ctxpropagation/worker/main.go

```diff
@@ -3,24 +3,30 @@ package main
 import (
 	"log"
 
-	"github.com/opentracing/opentracing-go"
+	"github.com/temporalio/samples-go/ctxpropagation"
```

Comment:
3 groups separated by empty line, please:
```go
"std imports"

"3rd party imports"

"this project local imports"
```

Reply:
:+1: Will change. I didn't realize this was by intention. I also haven't been requiring others do this in their contributions.

Reply:
Fixed